src/enrichment/lsst.rs

```diff
@@ -68,12 +69,99 @@ pub fn create_lsst_alert_pipeline() -> Vec<Document> {
     ]
 }
 
+pub struct Babamul {
+    kafka_producer: rdkafka::producer::FutureProducer,
+}
+
+impl Babamul {
+    fn new(config: config::Config) -> Self {
+        // Read Kafka producer config from kafka: producer in the config
+        // TODO: Do this properly
+        let kafka_producer_host = config
+            .get_string("kafka.producer")
+            .unwrap_or_else(|_| "broker:29092".to_string());
+
+        // Create Kafka producer
+        let kafka_producer: rdkafka::producer::FutureProducer =
+            rdkafka::config::ClientConfig::new()
+                // Uncomment the following to get logs from kafka (RUST_LOG doesn't work):
+                // .set("debug", "broker,topic,msg")
+                .set("bootstrap.servers", &kafka_producer_host)
+                .set("message.timeout.ms", "5000")
+                // it's best to increase batch.size if the cluster
+                // is running on another machine. Locally, lower means less
+                // latency, since we are not limited by network speed anyways
+                .set("batch.size", "16384")
+                .set("linger.ms", "5")
+                .set("acks", "1")
+                .set("max.in.flight.requests.per.connection", "5")
+                .set("retries", "3")
+                .create()
+                .unwrap();
+        Babamul { kafka_producer }
+    }
+
+    async fn process_alert(
+        &self,
+        alert: Document,
+        properties: Document,
+    ) -> Result<(), EnrichmentWorkerError> {
+        // Merge properties into the alert document
+        let mut merged = alert.clone();
+        merged.insert("properties", Bson::Document(properties.clone()));
+
+        // Determine if this alert is worth sending to Babamul
+        let min_reliability = 0.5;
+        let pixel_flags = vec![1, 2, 3];
```

Comment:
Just curious, is this one (on the pixel flags) just some kind of placeholder or some real cut you found somewhere?